fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -2835,7 +2845,7 @@ public void renameDatabase(AlterDatabaseRename stmt) throws DdlException {
             fullNameToDb.remove(fullDbName);
             fullNameToDb.put(newFullDbName, db);
 
-            DatabaseInfo dbInfo = new DatabaseInfo(fullDbName, newFullDbName, -1L);
+            DatabaseInfo dbInfo = new DatabaseInfo(fullDbName, newFullDbName, -1L, QuotaType.DATA);
```

Comment:
How about change `QuotaType.DATA` to `null`? For better understanding?

Reply:
@morningman null may cause NullPointerException, so I use QuotaType.NONE instead of null